core/jazz_codeq/index.js

```diff
@@ -173,14 +173,31 @@ function getReportOnError(err, metrics, config, serviceContext) {
 	let result = {};
 	if (err && err.report_error) {
 		if (err.code === 404) {
-			return utils.getReport(metrics, null, config)
+			logger.info('404 error'+err)
+		const result_inputs = getCodeqInputsUsingQuery(serviceContext, config);
```

Comment:
why are we making these changes. Can i get some context?

Reply:
these changes were not required,  have reverted it